.kitchen.yml

```diff
@@ -0,0 +1,32 @@
+---
+driver:
+  name: vagrant
+
+provisioner:
+  name: chef_zero
+  require_chef_omnibus: 12.9
+
+platforms:
+  - name: centos-7.2
+  - name: debian-8.4
+  - name: windows
+    driver:
+      box: opentable/win-2012r2-standard-amd64-nocm
+  - name: centos-7.2-compat
+    provisioner:
+      require_chef_omnibus: 12.4
+    driver:
+      box: bento/centos-7.2
+
+suites:
+  - name: install
+    run_list:
+      - recipe[cerner_splunk_ingredient_test::splunk]
+      - recipe[cerner_splunk_ingredient_test::universal_forwarder]
+  - name: uninstall
```

Comment:
49878a781929b6afc2aaa898a1b457fb6b50a11f

I added a recipe for universal forwarder and added it to the uninstall suite; is there any reason to split the uninstall suite for the two packages? There is no difference test-wise, and I intended to cut down on test time by not setting up chef omnibus on ~20 VMs.